CMakeLists.txt

```diff
@@ -53,7 +70,7 @@ target_compile_options(${LIBRARY} PUBLIC
   $<$<COMPILE_LANGUAGE:CXX>:-fno-rtti>
 
   -Wno-psabi
-  --specs=nosys.specs
+  -specs=/usr/lib/arm-none-eabi/lib/nosys.specs
```

Comment:
Does this work with a single hyphen? Be sure to test it with real programs that blink leds and things like that

Reply:
I have tested it with a nucleo and seems to work fine, also from the gcc page: 
![image](https://github.com/HyperloopUPV-H8/ST-LIB/assets/58850783/197e4876-8945-453d-9611-bc78c0073a07)